src/test/java/uk/gov/digital/ho/proving/income/validator/ProjectedAnnualIncomeCalculatorTest.java

```diff
@@ -98,11 +98,23 @@ public void thatRoundingErrorsAreHandled() {
         });
         aggregatedMonthlyIncome.put(12, new BigDecimal("1866.62"));
 
-        BigDecimal annualisedAverage = AnnualisedAverageCalculator.calculate(aggregatedMonthlyIncome);
+        BigDecimal projectedYearlyIncome = ProjectedAnnualIncomeCalculator.calculate(aggregatedMonthlyIncome);
 
-        assertThat(annualisedAverage).isEqualTo(new BigDecimal("22399.99"));
-        assertThat(annualisedAverage.scale()).isEqualTo(2);
+        assertThat(projectedYearlyIncome).isEqualTo(new BigDecimal("22399.99"));
+        assertThat(projectedYearlyIncome.scale()).isEqualTo(2);
     }
 
 
+    @Test
+    public void thatZeroIncomeMonhtsAreIgnored() {
```

Comment:
I know at the moment you are using `compareTo` method at the moment, but is it worth testing with zero BigDecimals with different decimal places?

Reply:
Hmmmm interesting thought.  So we are dealing in monetary amounts hence we always have 2dp so I'm not that interested in different decimal places.  However, abstracting the 2dp BigDecimal into its own class might be a good idea.  There is probably a Money class out in the wild somewhere.

I am going to ponder over this one for a bit.